### DISCUSSION TITLE
使用springboot启动在结束运行时会出现SimbotContext close handle 'MiraiRemoteResourceInProcessorHandle' execute failed!

### DISCUSSION THREAD

**Discussion by 1947406458** [ForteScarlet/simpler-robot#443] [不要在这里发帖子了]
使用springboot启动在结束运行时会出现SimbotContext close handle 'MiraiRemoteResourceInProcessorHandle' execute failed!

**Comment by ForteScarlet** [comment]
结束运行时会尝试关闭所有相关作用域，如果存在未完成任务则会出现此异常。

如果这种情况会导致预期外的结果，请通过issue反馈并提供应用场景、完整错误日志、复现手段与版本等详细信息。

如果有其他疑问，请描述您的问题。
